starrocks/benchmark.sh

```diff
@@ -2,18 +2,19 @@
 
 # This benchmark should run on Amazon Linux
 
-VERSION=3.0.0-preview
-DOWNLOAD_URL=https://releases.starrocks.io/starrocks/StarRocks-3.0.0-preview.tar.gz
+set -e
+
+VERSION=3.4.2-ubuntu-amd64
 # Install
-wget $DOWNLOAD_URL
+wget -q https://releases.starrocks.io/starrocks/StarRocks-$VERSION.tar.gz -O StarRocks-$VERSION.tar.gz
```

Comment:
Minor: I'll revert `-q` for consistency reasons (https://github.com/ClickHouse/ClickBench/pull/325)